Training/python/DataLoader.py

```diff
@@ -164,6 +171,7 @@ def get_predict_generator(self):
         '''
         assert self.batch_size == 1
         data_loader = R.DataLoader()
+        converter = torch_to_tf(return_truth=True, return_weights=False)
```

Comment:
I propose to add this args to get_predict_generator signature and read it from there

Reply:
Will also add this